agent/util-scripts/pbench-tool-meister-start

```diff
@@ -282,34 +284,77 @@ def main(argv):
         logger.exception("failed to load tool group data")
         return 1
 
+    # See if anybody told us to use certain options with SSH commands.
+    ssh_opts = os.environ.get("ssh_opts", "")
```

Comment:
Capitalize environment variables (I know there are a lot of legacy exceptions in the code, but...)

Reply:
Yes, this is an existing environment variable which is lower case, so can't change this.

Reply:
:ack: